fonts.css

```diff
@@ -1,66 +1,171 @@
 /* montserrat-regular - latin_cyrillic-ext_cyrillic */
+
 @font-face {
-    font-family: 'Montserrat';
-    font-style: normal;
-    font-weight: 400;
-    src: url('./fonts/montserrat-v14-latin_cyrillic-ext_cyrillic-regular.eot'); /* IE9 Compat Modes */
-    src: local('Montserrat Regular'), local('Montserrat-Regular'),
-    url('./fonts/montserrat-v14-latin_cyrillic-ext_cyrillic-regular.eot?#iefix') format('embedded-opentype'), /* IE6-IE8 */
-    url('./fonts/montserrat-v14-latin_cyrillic-ext_cyrillic-regular.woff2') format('woff2'), /* Super Modern Browsers */
-    url('./fonts/montserrat-v14-latin_cyrillic-ext_cyrillic-regular.woff') format('woff'), /* Modern Browsers */
-    url('./fonts/montserrat-v14-latin_cyrillic-ext_cyrillic-regular.ttf') format('truetype'), /* Safari, Android, iOS */
-    url('./fonts/montserrat-v14-latin_cyrillic-ext_cyrillic-regular.svg#Montserrat') format('svg'); /* Legacy iOS */
+  font-family: 'Montserrat';
+  font-style: normal;
+  font-weight: 400;
+  src: url('./fonts/montserrat-v14-latin_cyrillic-ext_cyrillic-regular.eot');
+
+  /* IE9 Compat Modes */
+
+  src:
+    local('Montserrat Regular'),
+    local('Montserrat-Regular'),
+    url('./fonts/montserrat-v14-latin_cyrillic-ext_cyrillic-regular.eot?#iefix') format('embedded-opentype'),
+            
```

Comment:
Codacy found an issue: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/Tesla_Project_AnastassiaVarabei_Evgen/pullRequest?prid=6131199)